kbfsgit/runner.go

```diff
@@ -258,8 +258,9 @@ func (r *runner) initRepoIfNeeded(ctx context.Context, forCmd string) (
 	}
 
 	// Only allow lazy creates for public and multi-user TLFs.
-	if r.h.Type() == tlf.Public ||
-		len(r.h.ResolvedWriters())+len(r.h.UnresolvedWriters()) > 1 {
+	numUsers := len(r.h.ResolvedWriters()) + len(r.h.UnresolvedWriters()) +
+		len(r.h.ResolvedReaders()) + len(r.h.UnresolvedReaders())
```

Comment:
It looks to me that an attempt by a reader to clone a nonexistent repo will get a `WriteAccessError`. Is that what we want?

Reply:
That was my plan, yeah.  Do you have another suggestion?

Reply:
Ideally it'd be the same error you get when a repo doesn't exist.

Reply:
Hmm, not sure I agree.  The repo will show up in the GUI if it exists, so it will be confusing if we say it doesn't exist.

Reply:
Right but it doesn't exist, and we failed to create it because you're a reader.

Reply:
It does exist from a Keybase perspective (i.e., `keybase git create` was run), it's just not initialized from a git perspective.

Reply:
Oh hmm...so you can `keybase git create` successfully when you're just a reader? That doesn't seem intuitive to me.

Reply:
No, I don't think you can do that.   I thought you were talking about the scenario where a writer made a repo, then a reader tried the URL before the writer initialized the repo.

Reply:
Ah, nope. I'm a reader, and I try to clone a repo that doesn't exist.

Reply:
Sorry, you said that pretty clearly and I missed it.

I'll see what I can do.

Reply:
Done: eb42e6a